inc/class-wp-forms-api.php

```diff
@@ -925,6 +927,29 @@ static function process_element( $element, &$values, &$input ) {
 	}
 
 	/**
+	 * Get WP editor templates for dynamically-inserted editors in #multiple fields
+	 *
+	 * @action wp_ajax_wp_form_get_mce_templates
+	 */
+	function ajax_get_mce_templates() {
+		$results = array();
+		$bodies = filter_input( INPUT_POST, 'content', FILTER_DEFAULT, FILTER_REQUIRE_ARRAY );
+
+		foreach ( $bodies as $id => $body ) {
+			ob_start();
+			$lines = preg_split( '/$/m', $body );
+
+			wp_editor( $body, $id, array(
+				'textarea_name' => $id,
+				'textarea_rows' => count( $lines ),
+			)  );
+			$results[$id] = ob_get_clean();
```

Comment:
I did a little research and couldn't find anything about the performance impact of starting and ending output buffering multiple times – the only difference that i see is that the output is potentially contained in memory slightly longer, but I think that happens with the PHP-level output buffering that is on by default anyway, and so it would just be a merely syntactical change that i think would be more confusing than having the start/get/end piece within the loop.